.github/actions/build-docs/action.yml

```diff
@@ -44,3 +57,4 @@ runs:
         mv documentation/_build /tmp/docs
         mv frontend/storybook-static /tmp/docs/storybook
         mv frontend/.tcv-export /tmp/docs/tailwind
+        mv api/api_docs /tmp/docs/api_docs
```

Comment:
This would run every time `main` is pushed, right? That would mean that the API docs would be updated to reflect _staging_ and be slightly ahead of production. Should we have a separate "staging_docs" area for pushes to `main` and then have the production docs built and published during Release app? :thinking: Originally I thought we could just copy the staging docs, but the production docs should reflect the code running in production which means it needs to handle rollbacks as well.

That's a big downside to including the API docs in the documentation site that I didn't consider before. That alone makes me doubt that this was a good suggestion (sorry!!!). Unless it's not too hard to juggle the documentation updates... What would we have to do? Each time the deployment workflow is dispatched, we'd need to pull the API image for the version being deployed, build the docs, then push them? We could put that into its own workflow that the deployment workflows dispatch but I'm starting to get dizzy thinking about that :dizzy_face: Maybe it wouldn't be so bad? What do you think? I don't think I'm making too much of the docs being ahead of production. Rollbacks are exceptional enough that we could ignore that case, at least for now... then all we'd have to do is rebuild the docs during Release app :thinking: hmmm...

Reply:
You're right @sarayourfriend. I've reverted the changes related to merging of the Redoc and the docs site, keeping this slightly more focused on the upgrade to `drf-spectacular`.